### DISCUSSION TITLE
Season of Docs 2022

### DISCUSSION THREAD

**Discussion by capsulecorplab** [nasa/fprime#1282] [General]
[Season of Docs](https://developers.google.com/season-of-docs) is a Google sponsored program that connects open-source organizations with technical writers to improve documentation. Applications for open-source organizations are open between February 23 - March 25 for 2022, as per their [timeline](https://developers.google.com/season-of-docs/docs/timeline). Could be a good opportunity for FPrime to validate its documentation.

**Comment by ThibFrgsGmz** [comment]
That sounds like a great idea!

**Comment by LeStarch** [comment]
I don't think our organization is eligible. 

https://developers.google.com/season-of-docs/terms/program-rules

"(c) not be a government agency or acting on behalf of a government or quasi-government agency that is funded by the public;"  the NASA affiliation invalidates us with respect to this statement.

**Reply by capsulecorplab** [reply]
Ah. I suppose that extends to the [F´ community](https://github.com/fprime-community/) organization as well?
